git-lfs_windows_arm64.go

```diff
@@ -0,0 +1,6 @@
+//go:build windows && arm64
+// +build windows,arm64
+
+//go:generate goversioninfo -arm=true
```

Comment:
The `-arm` flag needs to be defined explicitly when building for Windows ARM64. Support for this was added a few days ago: https://github.com/josephspurrier/goversioninfo/pull/51

Reply:
I don't have a better suggestion.  We practically never modify those files, so the burden of adding one more should be negligible.